packages/mobile/src/notifications/EscrowedPaymentReminderNotification.tsx

```diff
@@ -54,6 +59,7 @@ export class EscrowedPaymentReminderNotification extends React.PureComponent<Pro
         onPress: () => {
           const reclaimPaymentInput = payment
           CeloAnalytics.track(CustomEventNames.clicked_escrowed_payment_notification)
+          this.props.fetchReclaimSuggestedFee(payment.paymentID)
```

Comment:
This could take a few seconds to load. Does the confirmation card show a spinner in the meantime? Or what does it show?

Reply:
Since this function isn't making any real transactions, I've noticed it usually takes less than a second. However, should I add a spinner here just in case? If so, should I follow the SUCCESS/FAILURE action format which was discussed on the slack today?

Reply:
In that case, I'm okay to leave this as is for now